source/module_hamilt_lcao/module_deepks/LCAO_deepks.cpp

```diff
@@ -362,4 +382,56 @@ void LCAO_Deepks::del_orbital_pdm_shell(const int nks)
     return;
 }
 
+void LCAO_Deepks::init_v_delta_pdm_shell(const int nks,const int nlocal)
```

Comment:
Please add unit test for newly added functions.

Reply:
New integrate tests has covered the function.

Reply:
There are many newly added functions, they will reduce the coverage rate of unit tests.